docs/machine-learning/tutorials/iris-clustering.md

```diff
@@ -122,7 +122,7 @@ Add the following code to the `Main` method to setup the way to load data:
 
 [!code-csharp[Create text loader](~/samples/machine-learning/tutorials/IrisFlowerClustering/Program.cs#SetupTextLoader)]
 
-Note that the column names and indices match the schema defined by the `IrisData` class. The <xref:Microsoft.ML.Data.DataKind.R4?displayProperty=nameWithType> value specifies the `float` type.
+Use the [generic `CreateTextLoader`](/dotnet/api/microsoft.ml.textloadersavercatalog.createtextloader?view=ml-dotnet#Microsoft_ML_TextLoaderSaverCatalog_CreateTextLoader__1_Microsoft_ML_DataOperationsCatalog_System_Boolean_System_Char_System_Boolean_System_Boolean_System_Boolean_) method to infer the dataset schema from the `IrisData` class definition.
```

Comment:
Overall looks good. One minor change related to xref for `CreateTextLoader`. I have pasted the link below for the method being referenced in the tutorial.

[Microsoft.ML.TextLoaderSaverCatalog.CreateTextLoader%60%601
](https://xref.docs.microsoft.com/autocomplete?text=Microsoft.ML.TextLoaderSaverCatalog.CreateTextLoader%60%601)

Reply:
@luisquintanilla do you suggest to use the xref-link? I thought it would create a very long link text with all method parameters listed. Is that right? That might be overwhelming as most of the parameters are optional and only two are used in the sample. This is the reason I avoided the xref-link here.

Reply:
oh! I've realized that I still can provide the custom link text and use the xref-link; that way the display is short and the link is checked by the build.

@luisquintanilla thanks!